{"name": "Ошибка 403 при попытке создания дубликата пользователя", "status": "broken", "statusDetails": {"message": "NameError: name 'burger_api' is not defined", "trace": "self = <tests.test_create_user.TestCreateUser object at 0x00000176DCBA37A0>\n\n    @allure.title(\"Ошибка 403 при попытке создания дубликата пользователя\")\n    @allure.description(\"Создание дубликата пользователя. Проверка статуса и тела ответа\")\n    def test_duplicate_user_fail(self):\n>       body = burger_api.create_user_body()\nE       NameError: name 'burger_api' is not defined\n\ntests\\test_create_user.py:18: NameError"}, "description": "Создание дубликата пользователя. Проверка статуса и тела ответа", "start": 1737281673719, "stop": 1737281673720, "uuid": "27846a45-c4f5-44d5-ba05-2307d954b462", "historyId": "3b142cb6c5d6368ce0aa6d85b30d860f", "testCaseId": "3b142cb6c5d6368ce0aa6d85b30d860f", "fullName": "tests.test_create_user.TestCreateUser#test_duplicate_user_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "01ms32"}, {"name": "thread", "value": "8820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}